js/ui/scheduler/workspaces/view_model/utils.js

```diff
@@ -25,3 +25,35 @@ export const getViewDataGeneratorByViewType = (viewType) => {
             return new ViewDataGenerator();
     }
 };
+
+export function alignToFirstDayOfWeek(date, firstDayOfWeek) {
+    const newDate = new Date(date);
+    let dayDiff = newDate.getDay() - firstDayOfWeek;
+
+    if(dayDiff < 0) {
+        dayDiff += 7;
```

Comment:
The 7 looks obvious here, but let's move it to a separate constant.
Because it doesn't matter that it's obvious it's still the ["magic number"](https://www.inf.unibz.it/~calvanese/teaching/05-06-ip/lecture-notes/uni04/node17.html) from my point of view.